test/Microsoft.ML.Tests/ImagesTests.cs

```diff
@@ -710,5 +711,59 @@ public void TestBackAndForthConversionWithoutAlphaNoInterleaveNoOffset()
                 Done();
             }
         }
+
+        [Fact]
+        public void ImageResizerTransform_ResizingModeFill()
```

Comment:
>ImageResizerTransform_ResizingModeFill [](start = 20, length = 38)

Some of us have strong feelings regarding Python_style_of_defining_names, can you forgive us, and change it to C# style of `ImageResizerTransformResizingModeFill` ?

Reply:
No problem 😄